addons/website/views/snippets/s_kickoff.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_kickoff" name="Kickoff">
+    <section class="s_kickoff o_cc o_cc4 parallax s_parallax_is_fixed pt232 pb88 o_colored_level" data-oe-shape-data="{'shape': 'web_editor/Origins/14_001', 'colors': {'c3': 'o-color-4', 'c4': 'rgba(255, 0, 0, 0)'}, 'flip': [], 'showOnMobile': false}">
```

Comment:
The parallax option doesn't seem to be correctly defined:
![Capture d’écran 2024-08-26 à 09 01 05](https://github.com/user-attachments/assets/13b5e4c0-449e-442e-86cb-71b4d9bf26cb)
Maybe it's because `s_parallax_no_overflow_hidden` is not defined 🤔 Can you check ?